### DISCUSSION TITLE
[Feature] Ease the migration to WebP from Jpeg previews

### DISCUSSION THREAD

**Discussion by NyaomiDEV** [immich-app/immich#8980] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Right now, migration to WebP can be done by regenerating all thumbnails - easy enough, right? Yeah, but Immich won't delete the Jpeg counterpart once it's done converting to the shiny less space heavy WebP format! This causes disk usage issues especially on large image collections.

Therefore, I request a way to keep the thumbnail cache clean of duplicates, even if they are of different file formats. The format in the configuration shall be preferred and kept in case both are available in the cache.

(Also, I manually removed all the jpeg files in the cache and now I have some offline paths.. uhhhh....)

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by Ant-One** [comment]
Hi, I would like to switch from JPEG to WebP for my previews. Do you know if that's still the case in v2.7.5?

**Reply by bo0tzz** [reply]
Yes, this was resolved a long time ago but still open for some reason.

**Comment by bo0tzz** [comment]
#8981
